socioeconomic-differences/socioeconomic-differences.md

```diff
@@ -12,66 +12,88 @@ A person's socioeconomic status can inform their identity, as it affects the com
+     -However,it is the instructor task to emphasize that it is okay to feel so and simultaneously provide valuable solutions to the students.
 
 
 ### Guided Practice
 
-Have the apprentices work with you as you do something step-by-step.  This can also be fulfilled by a detailed tutorial intended for beginners.
+    Do not spend a lot of time explaining each point, rather try to fit the points effectively within the given time frame by explaining each points briefly but thoroughly,however try your best to include the suggestions and remarks made by students during the course of discussion.
+    1. The participants may feel reluctant to initially share their experience.The instructor should try to encourage them kindly and carefully without asserting any obligation.
+    2. The instructor can recount the examples of local professionals in the area,who too had blue-collar lifestyle but successfully transitioned to white-collar job.Also do mention the struggles they faced(however do not repeat or elaborate problems mentioned above, rather keep it more personal and healthy discussion).
 
 
 ### Independent Practice
+    
+	1.The students can further read articles,books,listen to podcasts or watch videos to enhance their understanding of the subject.
+    
+### Challenge (2 hrs research after the class,preferably done in group)
+     
+	1.Students can take a survey in their neighborhood(they can work in the group of 4), working places etc regarding people who identify themselves as "Class Straddlers" or people who have started working white-collar jobs after their previous blue-collar job and ask questions such as:-
+	a.  What is the percentage of increase in their salary after the shift in the job?
+	b.  What motivated them to change their profession?
+	c.  What are the major differences they feel with the transition of profession or lifestyle?
+  d.  Are they satisfied with the experience in their new work space?
+	e.  What are the set of new skills that they had to acquire to get the white-collar job?
+	f.  How did they learn the above mentioned skills?
+	(Further questions can be added as suitable)
+	The following data can be accumulated,with common reason binded together and thus the following data can be used to gain great deal of knowledge about the main issues,challenges and progress in their respective society.
+    The following collected data can also be used for further assignment or data-science project.
+
+	2.Students can work together to collect data to find which state in their country saw a major conversion of people from blue-collar job to white-collar job and what were the factors that was responsible, for e.g policies of their local government, increase of jobs due to sudden growth of offices in the areas, reason the offices were setup.  
+   
+### Check for Understanding(37-40 minutes)
+
+    1. A discussion can be held in the end where the participants summarize each other.
```

Comment:
Please change this section to specific topics to discuss in a large or small group.  No books, please. When you list the books in supplemental materials, maybe you can use these suggestions for why you think the book is a good resource.

Reply:
please move #2 to supplemental materials.

Reply:
please move #2 to supplemental materials.